docs/GETTING_STARTED.md

```diff
@@ -1,4 +1,4 @@
-# Contribution Guidelines
```

Comment:
This is named CONTRIBUTING as per GitHub specification https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors#adding-a-contributing-file